assets/src/edit-story/masks/frame.js

```diff
@@ -127,7 +139,14 @@ WithDropTarget.propTypes = {
   hover: PropTypes.bool,
 };
 
-export default function WithMask({ element, fill, style, children, ...rest }) {
+export default function WithMask({
+  element,
+  fill,
+  style,
+  children,
+  eventHandlers = null,
+  ...rest
```

Comment:
The reason why it's separated is that `...rest` is applied to the `div` below even if there is no mask set (the first `return`) -- then there will be a non-recognizable `eventhandlers` prop in the DOM. So we only need to add it in case of the latter returned `div`. Thoughts?

Reply:
But you're only setting the event handlers [in case of a mask being present](https://github.com/google/web-stories-wp/pull/5968/files/393064d0393c59a0654b1c1eaad5061c4946c5c6#diff-18261dd3f8a1ef6cc2042a471d3b8c1ce5ec9a81b23b4a52ce7df175dbde0a12R167), so the first return would never have them set anyway, no?

(also this "is mask disabled" calculation is copied out a few times)

Reply:
Yes, but the attribute will still be sent forward, even if with the value `null`.